roles/kubernetes-apps/istio/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: istio | Lay out manifests
+  template:
+    src: "{{item.file}}.j2"
+    dest: "{{kube_config_dir}}/{{item.file}}"
```

Comment:
can we add more level like
{{kube_config_dir}}/addons/istio

Reply:
I will refactor all addons and dirs in a followup

Reply:
I will refactor all addons and dirs in a followup